src/components/toolbar/README.md

```diff
@@ -0,0 +1,59 @@
+# MdToolbar
+`MdToolbar` is a vertical aligned bar, which can hold the application title or actions.
+
+### Screenshots
+![Preview](https://cloud.githubusercontent.com/assets/4987015/13727769/6d952c78-e900-11e5-890a-ccfd46996812.png)
+
+## `<md-toolbar>`
+### Bound Properties
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `color` | `"primary" | "accent" | "warn"` | The color palette for the toolbar |
+
+### Notes
+By the default the `md-toolbar` will use the background palette.
+
+Multiple rows inside of a `md-toolbar` can be added by appending as many as needed `<md-toolbar-row>` elements.
+
+### Example
+```html
+<md-toolbar [color]="myColor">
+  <span>First Toolbar Row</span>
+
+  <md-toolbar-row>
+    <span>Second Toolbar Row</span>
+  </md-toolbar-row>
+  
+  <md-toolbar-row>
+    <span>Third Toolbar Row</span>
+  </md-toolbar-row>
+</md-toolbar>
+```
+
+### Alignment
+The `md-toolbar` component doesn't provide any API to align the items probably in the row.
```

Comment:
I'd omit this line, and just go straight into saying that the toolbar content can be directly styles with flex, and then show some examples.